### ISSUE TITLE
Guide "How to Start" mit Pentacode

### ISSUE THREAD

**Issue by J0hhny** [pentacode/site#82]
Mir ist die Idee gekommen, dass ein "Start-Guide" sicherlich sinnvoll sein könnte für ganz neue Kunden.
Was hälst du davon? Vielleicht sogar schon erste Ideen was deiner Meinung unbedingt rein sollte?

**Comment by MaKleSoft** [comment]
Klar, wäre super sowas zu haben! Das Problem ist allerdings, dass sich das Onboarding von Anwender zu Anwender glaube ich etwas unterscheidet und deshalb nicht so leicht zu "formularisieren" ist. @PentacodeDennis @Thomasino21 Ihr könnt das vermutlich besser beurteilen. Was meint ihr?

**Comment by PentacodeDennis** [comment]
Grundsätzlich eine super Idee. 
Viele Neukunden nutzen die Software nicht zu 100% .
Und wir hätten eine große Zeitersparnis und könnten den Fokus auf andere Dinge richten.
